_posts/2017-01-26-create-an-fsharp-project-in-vscode.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+asset-type: post
+name: create-an-fsharp-project-in-vscode
+title: Create an F# project in VSCode
+date: 2017-01-26 10:00:00 +00:00
+author: Pedro Santos
+image:
+    src: /assets/img/custom/blog/2017-01-26-create-an-fsharp-project-in-vscode.jpg
+    attribution:
+      text: directions by Beat Tschanz
+      href: https://www.flickr.com/photos/btschanz/13793872333/
+tags:
+- F#
+- VSCode
+- Ionide 
+- Tutorial
+
+---
+
+## What we need:
+* Operating system  
+  * Windows
+  * MacOs
+  * Linux
+* CLR
+  * [.Net Framework](https://www.microsoft.com/net/download/framework) (Windows)
+  * [Mono](http://www.mono-project.com/]) (MacOs, Linux)
+* VSCode (https://code.visualstudio.com/)
+   * With ionide extensions for VSCode (http://ionide.io/)
+      * Ionide-FSharp
+      * Ionide-FAKE
+      * Ionide-Paket
+
+## How to step by step  
+1. Create new directory
+2. Move to new directory
+3. Type on console "Code ." or open VSCode and then open the new directory you just created
+4. Press shift-command-p and type F#
+5. Select New Project
+6. Choose classlib or console or other project type for the production project
+7. Choose the project directory (if empty selects current directory)
+8. Provide a project name (using production.fsproj for this example)
+9. Press shift-command-p and type FAKE
+10. Chose Default build, you should see an output similar to this:
+
+```txt
+Checking Paket version (downloading latest stable)...
+Paket.exe 3.33.5 is up to date.
+Paket version 3.33.5
+0 seconds - ready.
+Building project with version: LocalBuild
+Shortened DependencyGraph for Target Build:
+<== Build
+   <== Clean
+...
+```
+11. shift-command-p and type F#
+12. Select New Project
+13. Choose fsunit for test code
+14. Choose the project directory (empty)
+15. Provide a project name (using test.fsproj for this example)
+16. Open up the test.fsproj file
+17. Press shift-command-p and type F#
+18. Select Add Project Reference
+19. Choose test.fsproj project as the project that you want to edit
+20. Choose production.fsproj project as the reference you want to add
+21. Verify that test.fsproj has been changed and contain reference to production.fsproj
```

Comment:
Verify that `test.fsproj` has been changed and contains a reference to `production.fsproj`